lib/govuk_publishing_components/presenters/page_with_step_by_step_navigation.rb

```diff
@@ -62,6 +63,11 @@ def header
       attr_reader :content_item, :current_path
 
       def first_step_nav
+        if @active_step_nav_content_id
```

Comment:
This should `@active_step_nav_content_id.present?` as you are checking if the value exists.